docs/src/main/asciidoc/mongodb.adoc

```diff
@@ -676,9 +676,39 @@ This means that the `org.acme.MyVariable` class is not known to GraalVM, the rem
 More details about the `@RegisterForReflection` annotation can be found on the xref:writing-native-applications-tips.adoc#registerForReflection[native application tips] page.
 ====
 
-== Conclusion
+== Using mongo+srv:// urls
 
-Accessing a MongoDB database from a MongoDB Client is easy with Quarkus as it provides configuration and native support for it.
+`mongo+srv://` urls are supported out of the box in JVM mode.
+However, in native, the default DNS resolver, provided by the MongoDB client, uses JNDI and does not work in native mode.
+
+If you need to use `mongo+srv://` in native mode, you can configure an alternative DNS resolver.
+This feature is **experimental** and may introduce a difference between JVM application and native application.
+
+To enable the alternative DNS resolver, use:
+
+[source, properties]
+----
+quarkus.mongodb.native.dns.use-vertx-dns-resolver=true
+----
+
+As indicated in the property name, it uses Vert.x to retrieve the DNS records.
+By default, it tries to read the first `nameserver` from `/etc/resolv.conf`, if this file exists.
+You can also configure your DNS server:
+
+[source properties]
+----
+quarkus.mongodb.native.dns.use-vertx-dns-resolver=true
```

Comment:
I wonder if I would have used the `native` prefix for the config. Your plan is to allow using this in both JVM and native at some point, right?

I agree it's more explicit now but might be a bit confusing later. But maybe deprecating them once we have a common behavior is OK.

Reply:
While it's the plan, it's hard to tell if it will ever be possible (it's not on us but on mongo). This is why I made "native" explicit. Now, I don't have a strong opinion, I just want to avoid confusion. Maybe mentioning "native" in the documentation is enough (but if one day we can do it for both, wouldn't his be also confusing?)